package-manifest.toml

```diff
@@ -153,6 +153,7 @@ source.type = "composite"
 source.packages = [
   "clickhouse_svc.tar.gz",
   "internal-dns-cli.tar.gz",
+  "omicron-clickhouse-admin.tar.gz",
```

Comment:
I don't think we actually want to add it here. There's the new `ClickhouseServer` omicron zone where it should probably live, although we don't yet deploy that. Also fine to leave this for now and take it out later to help testing.

Reply:
Same as above it was meant for testing, left a comment to specify this :)